pkg/gadget-service/service.go

```diff
@@ -142,6 +142,7 @@ func (s *Service) RunGadget(runGadget api.GadgetManager_RunGadgetServer) error {
 	if err != nil {
 		return fmt.Errorf("initialize operators: %w", err)
 	}
+	defer operators.GetAll().Close()
```

Comment:
Operators are lazily loaded here, but they shouldn't be closed unless the service process is stopped (or we find a good way to schedule `.Close()` calls after a certain timeout to free resources) as other running gadgets might still use them.

Reply:
I remembered so too, that `Init()` and `Close()` is shared between all of currently running gadgets.

But then I saw this `Init()` which is called for every gadget run and it confused me that I don't see the same amount of `Close()` for that.

Reply:
I removed the commit containing the `close` since the other commit is one I want to merged as soon as possible